.gitignore

```
@@ -30,3 +30,5 @@ tags
 /stamp-h1
 /cov-int
 /libseccomp-coverity_*.tar.gz
+html
+latex
```

This will ignore all directories (or files) named `html` or `latex` which may cause issues down the road.

If you want to ignore the top-level directories, it's better to write this as
```suggestion
/html/
/latex/
```

---

Good call.  Thanks, @kolyshkin